understanding/22/dragging-movements.html

```diff
@@ -41,7 +41,7 @@ <h2>Intent of Dragging Movements</h2>
 
     <p>This Success Criterion applies to dragging motions as opposed to pointer gestures, which are covered in Success Criterion 2.5.1 Pointer Gestures. Pointer gestures include directional path-based as well as multi-point gestures. In contrast, for dragging motions, only the start and end point of the motion matters, not the actual path.</p>
 
-      <p>There may be cases where a dragging movement is essential, for example, drawing a freeform figure, or playing a game where the aim is to follow a particular track as accurately as possible. The criterion has an exception for these cases where the task requires dragging.</p>
+      <p>There may be cases where a dragging movement is essential, for example, when playing a game where the aim is to quickly drag an element over a moving, jumping, or suddenly emerging dragging target. The criterion has an exception for these cases where the task requires dragging.</p>
```

Comment:
`dragging target` maybe better `drop target` (to keep with the drag-n-drop nomenclature for things)